docs/project/contribution_tools.md

```diff
@@ -130,8 +130,9 @@ These tools are essential for work on Carbon.
             outdated, and not be upgraded.
 -   Main tools
     -   [Bazel](https://www.bazel.build/)
+        -   At least bazel version 6.0.0.
```

Comment:
```suggestion
        -   NOTE: See [the bazelisk config](/.bazeliskrc) for a supported version.
```

Thanks for pointing this out. We have this in a config file, how about pointing at that if someone needs a specific version? That way, we're less likely to forget to update a spot if the advice changes.